magento/inventory
# MSI-896: Disabled "Advanced Inventory" button.

### Description
Makes "Advanced Inventory" button stay active in any case, even when "Manage Stock" = "No".

### Fixed Issues (if relevant)
1. magento-engcom/msi#896: Disabled "Advanced Inventory" button.

### Manual testing scenarios
1. Go to the product creation page.
2. Press "Advanced Inventory" button.
3. In modal window set "Manage Stock" to "No", press "Done".
4. Make sure "Advanced Inventory" button stays active.
5. Press "Advanced  Inventory" button.
6. In modal window set "Manage Stock" to "Yes". Press "Done".
7. Make sure "Advanced Inventory" button stays active and "Assign Sources" button appears on page and also active.
